### DISCUSSION TITLE
Now Playing Text

### DISCUSSION THREAD

**Discussion by tmiradio** [AzuraCast/AzuraCast#6673] [General]
I have two channels on my radio station and I am looking at  having a text html script to show what is playing currently on both server channels. 
Are you able to assist with this matter?

**Comment by LordHelmchen666** [comment]
Look here https://www.azuracast.com/docs/developers/now-playing-data/ or https://github.com/AzuraCast/AzuraCast/discussions/3576#discussioncomment-5824323

**Comment by tmiradio** [comment]
I am having a hard time trying to understand all this and what I would need to change in order to show the now playing information on my station.

**Comment by Moonbase59** [comment]
I’d say have a look here → https://github.com/AzuraCast/AzuraCast/discussions/6762#discussioncomment-7718655 but this is even more complex… (albeit the right way to do it).

Maybe using one of AzuraCast’s Embed Widgets (in an iframe) would be an easier way for you to use?
